Bottom: 62334c9976f257e1caad9dda66e21545e2f76217
Top:    2401eb4379a5c8c183fc97a7a2d46fa2ad42770c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 14:23:53 +0000

Refresh of add-polygon-parentage-tree

---

diff --git a/src/polygon.c b/src/polygon.c
index 87959a2..4806c08 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -989,7 +989,6 @@ SubtractPad (PadType * pad, PolygonType * p)
           (np = LinePoly ((LineType *) pad, pad->Thickness + pad->Clearance)))
         return -1;
     }
-
   return Subtract (np, p, true);
 }
